changelog.md

```diff
@@ -0,0 +1,61 @@
+# Changelog
+
+All the notable changes and updates
+
```

Comment:
a mention that we are following semantic versioning, with a link to the changelog strat at [https://keepachangelog.com/en/1.0.0/]

Reply:
Looks amazing, excited to start working with it. You should include a mention of semantic versioning, and maybe, a link to the logging strat at https://keepachangelog.com/en/1.0.0/, or just a brief mention of how you want developers to log.